### DISCUSSION TITLE
how to loop a command?

### DISCUSSION THREAD

**Discussion by JorySeverijnse** [nushell/nushell#11601] [Q&A] [answered]
how do we loop a command in nushell, i've done it before in nushell but i dont remember how. this is what i tried
while true {ps; sleep 1sec}
loop {ps; sleep 1sec}

when i echo something it does work normally how i would expect it to

**Comment by fdncred** [comment, marked as answer]
```nushell
while true {echo (ps); sleep 1sec}
loop {echo (ps); sleep 1sec}
```

**Reply by JorySeverijnse** [reply]
makes sense, thanks for the fast answer :)
